clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java

```diff
@@ -1641,6 +1641,12 @@ private static boolean isClonedStoreOnline(
       int srcLatestOnlineVersionOfMetaSystemStore = getLatestOnlineVersionNum(srcMetaSystemStore.getVersions());
       int destLatestOnlineVersionOfMetaSystemStore = getLatestOnlineVersionNum(destMetaSystemStore.getVersions());
       destMetaStoreOnline = destLatestOnlineVersionOfMetaSystemStore >= srcLatestOnlineVersionOfMetaSystemStore;
```

Comment:
Is this check correct? I am wondering if original cluster repush meta store before and has v3 online, then this condition will never become true right?

Reply:
The code was added in 2021 and no issue so far. If original meta system store version 3, then version 3 will be added to the destination meta system store. See code here https://github.com/linkedin/venice/blob/d5131fd382d8d102f4a89580c2ec81dd13fa0ace/services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java#L1358

As destination meta system store has version 3, which >= original meta system store version, the condition will be true.